tests/unit/panther_analysis_tool/test_main.py

```diff
@@ -35,6 +36,9 @@
 
 print('Using fixtures path:', FIXTURES_PATH)
 
+if os.environ.get('PANTHER_ANALYSIS_TOOL_DEBUG'):
```

Comment:
Is this a env variable you are setting manually? Is the idea to set this env variable _or_ use the `--debug` command line option?

Reply:
Well, that's only for the tests, but I think I'll remove it altogether and maybe revisit in a different PR. It is generally useful to activate debug-level logging, in case something breaks, as it is sometimes difficult to find the root cause for the high-level test failures.